be/src/block_cache/block_cache.cpp

```diff
@@ -168,8 +168,8 @@ void BlockCache::record_read_cache(size_t size, int64_t lateny_us) {
     _kv_cache->record_read_cache(size, lateny_us);
 }
 
-const DataCacheMetrics BlockCache::cache_metrics() const {
-    return _kv_cache->cache_metrics();
+const DataCacheMetrics BlockCache::cache_metrics(int level) const {
```

Comment:
can we use enum instead?

Reply:
> can we use enum instead?

It is mainly to be consistent with the dependent library interface, and we can replace it after the library is updated.